docs/Analyzer Configuration.md

```diff
@@ -741,3 +741,15 @@ Configurable Rules: [CA1711](https://docs.microsoft.com/visualstudio/code-qualit
 Option Values: List (separated by '|') of allowed suffixes
 
 Example: `dotnet_code_quality.CA1711.allowed_suffixes = Flag|Flags`
+
+### Enable platform compatibility analyzer for TFMs <= net5.0
+
+Option Name: `enable_platform_analyzer`
```

Comment:
Given the option is only applicable to pre-.NET5 TFMs, maybe we can make it explicit in the option name so it is clear that the option is not applicable to net5 or later TFMs? If you make this change, please update all uses of `enable_platform_analyzer` in this PR.
```suggestion
Option Name: `enable_platform_analyzer_on_pre_net5`
```

Reply:
Yeah i have considered naming something like that and thought it is a too long name for devs to memorize, but after looking naming for other options seems we try to give a more descriptive name than keeping it short, will update